x-pack/plugins/observability/public/pages/alerts/components/render_cell_value/render_cell_value.tsx

```diff
@@ -38,6 +38,17 @@ export const getMappedNonEcsValue = ({
   return undefined;
 };
 
+const getRenderValue = (mappedNonEcsValue: any) => {
+  // can be updated when working on https://github.com/elastic/kibana/issues/140819
+  const value = Array.isArray(mappedNonEcsValue) ? mappedNonEcsValue[0] : mappedNonEcsValue;
+
+  if (!value || typeof value === 'string' || typeof value === 'number') {
```

Comment:
I think this code is related to this bug https://github.com/elastic/kibana/pull/140516#issuecomment-1248661869 when I changed to something simpler everything start to works by using isEmpty from `lodash`
```suggestion
  if (!isEmpty(value)) {
```